dev-lisp/newlisp/newlisp-10.7.5.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="Put the fun back in Lisp"
+DESCRIPTION="newLISP is a LISP-like scripting language for doing things"
+HOMEPAGE="https://www.newlisp.org"
+COPYRIGHT="2013 Lutz Mueller"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://www.newlisp.org/downloads/newlisp-10.7.5.tgz"
+SOURCE_FILENAME="newlisp-$portVersion.tar.gz"
+CHECKSUM_SHA256="dc2d0ff651c2b275bc4af3af8ba59851a6fb6e1eaddc20ae75fb60b1e90126ec"
+PATCHES="newlisp-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	newlisp$secondaryArchSuffix = $portVersion
+	cmd:newlisp = $portVersion
+	cmd:newlisp_$portVersion = $portVersion
+	cmd:newlispdoc = $portVersion
+	"
+
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libffi$secondaryArchSuffix
+	lib:libncursesw$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libffi$secondaryArchSuffix
+	devel:libncursesw$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+BUILD()
+{
+	runConfigure ./configure-alt
+	make
+}
+
+INSTALL()
+{
+	mkdir -p $binDir
```

Comment:
Not needed, done by make install, and in this case with `$commandBinDir `it would be `$prefix/bin`, not `$binDir `which creates `$prefix/bin/x86` on 32bit secondary architecture.
```diff
- 	mkdir -p $binDir
```

Reply:
removed

Reply:
removed